ci/integration/logging/run_bridge.sh

```diff
@@ -5,14 +5,8 @@
 set -ex
 
 # enable required repo(s)
-cat > /etc/yum.repos.d/fedora-eln.repo <<EOF
-[centos-opstools]
-name=opstools
-baseurl=http://mirror.centos.org/centos/8/opstools/\$basearch/collectd-5/
-gpgcheck=0
-enabled=1
-module_hotfixes=1
-EOF
+curl -o /etc/yum.repos.d/CentOS-OpsTools.repo $OPSTOOLS_REPO
```

Comment:
the repo file is downloaded? Why is it stored as well? Can it be either curled twice, or not downloaded at all?

Reply:
It seems like if the GHA did the file pull, it could be made available from a single location as a mounted file in the build container, or something like that. Having the container image itself pulling the file into its system locally seems unnecessary.

Reply:
Stored for build, fetched for CI jobs. Builds should be more stable, CI jobs more change agnostic ... IMO